misc/futures-bounded/src/lib.rs

```diff
@@ -0,0 +1,193 @@
+use std::future::Future;
+use std::task::{Context, Poll, Waker};
+use std::time::Duration;
+
+use futures_timer::Delay;
+use futures_util::future::{select, BoxFuture, Either};
+use futures_util::stream::FuturesUnordered;
+use futures_util::{ready, FutureExt, StreamExt};
+
+/// Represents a set of (Worker)-[Future]s.
+///
+/// This wraps [FuturesUnordered] but bounds it by time and size.
```

Comment:
```suggestion
/// This wraps [`FuturesUnordered`] but bounds it by time and size.
```

Reply:
I think this still renders as a link but I'll try!

Reply:
Works:

![image](https://github.com/libp2p/rust-libp2p/assets/5486389/f0cc341f-7434-455b-8ac6-598bfded9b95)

Reply:
Don't you want `FuturesUnordered` to be monospaced to highlight, that it is a symbol?